packages/amplify-cli/src/domain/amplify-usageData/UsageData.ts

```diff
@@ -57,16 +61,31 @@ export class UsageData implements IUsageData {
     this.input = redactInput(input, true);
     this.codePathTimers.set(FromStartupTimedCodePaths.PLATFORM_STARTUP, Timer.start(processStartTimeStamp));
     this.codePathTimers.set(FromStartupTimedCodePaths.TOTAL_DURATION, Timer.start(processStartTimeStamp));
+    UsageData.flow.setInput(input);
+    UsageData.flow.setVersion(version);
   }
 
   /**
    * Get the usage data singleton
    */
   static get Instance(): IUsageData {
-    if (!UsageData.instance) UsageData.instance = new UsageData();
+    if (!UsageData.instance) {
+      UsageData.instance = new UsageData();
+      UsageData.flow = CLIFlowReport.instance;
+    }
     return UsageData.instance;
   }
 
+  /**
+   * Get the flow data singleton
+   */
+  static get flowInstance(): IFlowData {
```

Comment:
This is an artifact of an earlier discussion where we eventually wanted to separate the two paths (UsageMetrics and FlowLogs) and use a different API/ingestion engine. It totally makes sense to remove this since the interface is already implemented by UsageData.